.mailmap

```diff
@@ -245,6 +249,7 @@ Shanggao Qiu <qiushanggao@qq.com> qiushanggao
 Shawn Chen <cxwshawn@gmail.com> <cxwshawn@gmail.com>
 Shu, Xinxin <xinxin.shu@intel.com> xinxin shu <xinxin.shu@intel.com>
 Shu, Xinxin <xinxin.shu@intel.com> xinxinsh <xinxin.shu@intel.com>
+Song Baisen <song.baisen@zte.com.cn>
```

Comment:
Does an entry like this change the output or so.. As far the author has used only one email, I guess this entry could be dropped